core/modules/ccontrol/queryMsg.cc

```diff
@@ -43,19 +44,23 @@ int queryMsgGetCount(int session) {
 }
 
 // Python call: msg, chunkId, code, timestamp = queryMsgGetMsg(session, idx)
-std::string queryMsgGetMsg(int session, int idx, int* chunkId, int* code, time_t* timestamp) {
+std::string queryMsgGetMsg(int session, int idx, int* chunkId, int* code, lsst::qserv::MessageSeverity* severity, time_t* timestamp) {
 
```

Comment:
I'd shorten this line - can you put last arg in separate line? Also, you don't need "last::qserv::" in front (revisit this part in other places too)